packages/react/.storybook/main.ts

```diff
@@ -25,6 +25,10 @@ const config: StorybookConfig = {
     options: {},
   },
   staticDirs: ["../src", "../public"],
+  managerHead: (head) => `
```

Comment:
Couldn't you use the existing `preview-head.html`? 🤔

Reply:
`preview-head` template is used to populate the head of iframe preview blocks not the head of the page document.

But I realized this code is useless as the head already contains this tag : 

`<link rel="icon" type="image/svg+xml" href="./favicon.svg">`

Just the fact to put favicon.svg into public folder is enough.